relay-general/tests/snapshots/test_fixtures__event_schema.snap

```diff
@@ -1427,15 +1427,56 @@ expression: "relay_general::protocol::event_json_schema()"
     },
     "EventType": {
       "description": "The type of an event.\n\nThe event type determines how Sentry handles the event and has an impact on processing, rate limiting, and quotas. There are three fundamental classes of event types:\n\n- **Error monitoring events** (`default`, `error`): Processed and grouped into unique issues based on their exception stack traces and error messages. - **Security events** (`csp`, `hpkp`, `expectct`, `expectstaple`): Derived from Browser security violation reports and grouped into unique issues based on the endpoint and violation. SDKs do not send such events. - **Transaction events** (`transaction`): Contain operation spans and collected into traces for performance monitoring.",
-      "type": "string",
-      "enum": [
-        "error",
-        "csp",
-        "hpkp",
-        "expectct",
-        "expectstaple",
-        "transaction",
-        "default"
+      "oneOf": [
```

Comment:
cargo bumped schemars, and as a result schemars apparently decided to make this change to the event schema. i think the change is harmless, but we can probably also just pin the direct schemars dep to a different version